packages/pn-commons/src/models/NotificationDetail.ts

```diff
@@ -219,6 +219,7 @@ export interface PaidDetails extends BaseDetails {
   creditorTaxId: string;
   noticeCode: string;
   uncertainPaymentDate?: boolean;
+  eventTimestamp?: string;
```

Comment:
Check the open api on delivery. This field maybe is for all the details and can be moved into BaseDetails

Reply:
Ok, I moved it into BaseDetails because it is also used in `NotificationViewedCreationRequestDetailsV23`, `NotificationRADDRetrievedDetails`, `RefinementDetailsV23`, `SendDigitalProgressDetailsV23` and `NotificationPaidDetailsV23`